src/tests/Rockets.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import Rockets from '../components/Rockets';
+import { reserveRocket } from '../redux/rockets/rocketsSlice';
+
+const mockStore = configureStore([]);
+
+describe('Rockets Component', () => {
+  let store;
+
+  beforeEach(() => {
+    const initialState = {
+      rockets: {
+        rockets: [
+          {
```

Comment:
- [ ]  You did a great job, but consider fixing this issue I am facing you tests to proceed  and implement the previous reviewer request properly. 
![image](https://github.com/davidolanrewaju/Space-Traveler-Hub/assets/43877689/c37fb137-8d7a-4cd6-99ad-89caee8506bc)